user/client_test.go

```diff
@@ -422,3 +422,22 @@ func TestUserClientListOrganizationMemberships(t *testing.T) {
 	require.Equal(t, organizationID, list.OrganizationMemberships[0].Organization.ID)
 	require.Equal(t, userID, list.OrganizationMemberships[0].PublicUserData.UserID)
 }
+
+func TestUserClientDeletePasskey(t *testing.T) {
+	t.Parallel()
+	userID := "user_123"
+	passkeyIdentificationID := "idn_345"
+	config := &clerk.ClientConfig{}
+	config.HTTPClient = &http.Client{
+		Transport: &clerktest.RoundTripper{
+			T:      t,
+			Out:    json.RawMessage(fmt.Sprintf(`{"id":"%s"}`, passkeyIdentificationID)),
+			Method: http.MethodDelete,
+			Path:   "/v1/users/" + userID + "/passkeys/" + passkeyIdentificationID,
+		},
+	}
+	client := NewClient(config)
+	passkey, err := client.DeletePasskey(context.Background(), userID, passkeyIdentificationID)
```

Comment:
❓ Do we need to seed a passkey?
What happens if we call `client.DeletePasskey` twice in this test?

Reply:
the test only asserts the http call for BAPI through the transporter. Defined [here](https://github.com/clerk/clerk-sdk-go/blob/3445b5e9a20ceca1f1317c70bf21cbcf81667dee/user/client_test.go#L432-L437)

Reply:
I see, so `Out` is effectively the mocked-expected-response